EIPS/eip-3085.md

```diff
@@ -7,7 +7,7 @@ status: Draft
 type: Standards Track
 category: Interface
 created: 2020-11-01
-requires: 155, 695, 3091
+requires: 155, 695, 2696, 3091
```

Comment:
While not changed in this PR, be aware that 3091 is still draft and thus will prevent this EIP from moving forward.  You may want to reach out to the author of that EIP and see what their plans are for getting that moved to review and eventually final.